content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Secure Supply Chain: Verifying image signatures"
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+After these last releases Kubewarden now has support for verifying the integrity and authenticity of artefacts within
+Kubewarden using the Sigstore project. In this post, we shall focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [post](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Pods by checking their container images for signatures (that is, containers, init containers and 
+ephemeral containers in the pod). If all signature validations pass or there is no container image that matches the image
+name query, the Pod will be accepted.
+
+Once the image has been verified with a signature, one needs to ensure that the specific image tag from that signature 
+is the one instantiated. This is achieved by the policy by mutating the Pod request and adding that validated image digest. 
```

Comment:
I also agree with Jose. On top of that, I think something like that would be maybe better:

```suggestion
Container image tags are mutable, they can be changed to point to a completely different content. That's why it's a
good security practice to reference container images by their immutable checksum.

This policy can rewrite the image definitions that are using a tag to instead reference the image by its checksum.

The policy will:
  * Ensure the image referenced by a tag is satisfying the signature requested by the operator
  * Extract the immutable reference of the image from the signatures
  * Rewrite the image reference to be in the form `<image ref>@sha256:<digest>`
```